lib/src/components/help_option_button.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+/// [HelpOptionButton] uses for actions
+class HelpOptionButton extends StatefulWidget {
+  /// A variable that is used to set the height of the button.
+  final double height;
+
+  /// A variable that is used to set the title of the button.
+  final String title;
+
+  /// A callback for the button.
+  final VoidCallback? onTap;
+
+  /// Used to set the alignment of the button.
+  final bool isExpandedCenter;
+
+  /// A variable that is used to set the icon of the button.
+  final IconData icon;
+
+  /// A variable to set a bacground color for button;
+  final Color? backgroundColor;
+
+  /// A variable that is used to set the color of the button when the mouse is
+  /// hovering over it.
+  final Color? hoverColor;
+
+  /// A variable that is used to set the size of the icon.
+  final double iconSize;
+
+  ///Constructor
+  const HelpOptionButton({
+    Key? key,
+    this.height = 51,
+    required this.title,
+    this.onTap,
+    this.isExpandedCenter = false,
+    required this.icon,
+    this.iconSize = 14,
+    this.backgroundColor,
+    this.hoverColor,
```

Comment:
Let's push `required` pieces to the top:
```suggestion
    required this.icon,
    required this.title,
    Key? key,
    this.height = 51,
    this.onTap,
    this.isExpandedCenter = false,
    this.iconSize = 14,
    this.backgroundColor,
    this.hoverColor,
```